pages/project/[[...slug]]/index.js

```diff
@@ -321,29 +321,17 @@ function GraphTooltip({ repository, tooltipData }) {
 
 function Metrics({ id, repository }) {
   const router = useRouter()
-  const { metric: queryKey } = router.query
+  const [projectSlug, metricSlug] = router.query.slug
   const { data: metrics } = useSWR('/api/metrics/' + id, fetcher)
   const [selectedMetric, setSelectedMetric] = useState()
```

Comment:
Can we remove this state entirely? (which also means there's no need to have an effect.

Reply:
In use it's unclear, this state is just a duplication of the url argument, so it doesn't need to exist at all. `selectedMetric` can be computed using `metrics` and `metricSlug`